src/translitWordMap.js

```diff
@@ -0,0 +1,29 @@
+/* load modules */
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Returns a map of words to be used in a word-by-word transliteration
+ *
+ * @since 1.0.0
+ * @param {string} filePath The path to the csv file representing the word map
+ * @returns {Map} Returns a Map object representation of the input csv file
+ */
+
+function translitWordMap(filePath) {
```

Comment:
better name (for the file as well) would be `readCsvWordMap`. is this just for the test or do we expect user's to call this (not likely IMO).

Reply:
if its just for the test then I'd remove.

Reply:
Its a user callable function by design; the idea is to provide a utility for the user to generate a map from a simple csv file, and also abstract this functionality. Performance-wise, the best practice would be the user generates a map once in their script and then passes it in wherever the translit function is called.

Reply:
I'm not sure if its a good idea to except the user to have their own Javascript Map object ready to use with anvaad.